src/templates/layouts/healthCareLocalFacility/index.test.tsx

```diff
@@ -24,4 +32,16 @@ describe('HealthCareLocalFacility with valid data', () => {
       expect(screen.getByText(mockData[key])).toBeInTheDocument()
     )
   })
+
+  // Once window.sideNav is populated, the static-pages app will render the menu
+  // in the nav[data-widget-type="side-nav"] element, but testing for that
+  // render would require side-loading that app bundle, which is outside the
+  // scope of this unit test
```

Comment:
`static-pages` is built in `vets-website`, so including it for this test is non-trivial. If we care deeply, I can figure out how to make that happen in a follow-up, though.